modules/slugs/includes/qtranslate-slug-settings-options.php

```diff
@@ -47,15 +47,27 @@ function qts_options_page_fields() {
         $options[] = qts_options_page_build_slug_fields( $taxonomy, "taxonomies", "taxonomy_" );
     }
 
-    return $options;
+    return array_filter($options);
```

Comment:
What is that for?

Reply:
if `rewrite==false` for a post/taxonomy type, empty array is added in `qts_options_page_build_slug_fields` to `$options`, just cleaning up the array here.